pkg/watchdogd/watchdogd.go

```diff
@@ -7,42 +7,69 @@
 //
 // It starts in the running+armed state:
 //
-//             \| watchdogd Running     | watchdogd Stopped
+//              | watchdogd Running     | watchdogd Stopped
 //     ---------+-----------------------+--------------------------
 //     Watchdog | watchdogd is actively | machine will soon reboot
 //     Armed    | keeping machine alive |
 //     ---------+-----------------------+--------------------------
 //     Watchdog | a hang will not       | a hang will not reboot
 //     Disarmed | reboot the machine    | the machine
 //
-// The following signals control changing state:
-//
-//     - STOP: running -> stopped
-//     - CONT: stopped -> running
-//     - USR1: armed -> disarmed
-//     - USR2: disarmed -> armed
+
 package watchdogd
 
 import (
 	"context"
+	"errors"
 	"fmt"
-	"io/ioutil"
+	"io"
 	"log"
+	"net"
 	"os"
-	"os/signal"
-	"path/filepath"
-	"strconv"
 	"strings"
 	"time"
 
 	"github.com/u-root/u-root/pkg/watchdog"
 	"golang.org/x/sys/unix"
 )
 
-// DaemonProcess is the name of the daemon process.
-const DaemonProcess = "watchdogd"
+const defaultUDS = "/tmp/watchdogd"
+
+const (
+	OpStop     = 'S' // Stop the watchdogd petting.
+	OpContinue = 'C' // Continue the watchdogd petting.
+	OpDisarm   = 'D' // Disarm the watchdog.
+	OpArm      = 'A' // Arm the watchdog.
+)
+
+const (
+	OpResultOk        = 'O' // Ok.
```

Comment:
note for future: it's nice that there are no command values that also are an error value.

Reply:
noted